packages/esm-tb-app/src/views/patient-summary/tb-patient-summary.component.tsx

```diff
@@ -124,7 +130,12 @@ const TBSummaryOverviewList: React.FC<PatientChartProps> = ({ patientUuid }) =>
         key: 'regimen',
         header: t('regimen', 'Regimen'),
         getValue: (encounter) => {
-          return getObsFromEncounter(encounter, config.obsConcepts.regimen);
+          const tBEnrollmentType = findObs(encounter, config.obsConcepts.tBEnrollmentType)?.value?.uuid;
```

Comment:
Can you encapsulate the logic of getting the regimen in a function and replace in both occurences?

Reply:
Done